target/linux/realtek/dts-5.10/rtl839x.dtsi

```diff
@@ -87,6 +97,13 @@
 		#size-cells = <1>;
 		ranges = <0x0 0x18000000 0x10000>;
 
+		ccu: clock-controller {
```

Comment:
This device's memory is not in the range covered by `/soc`. Since we don't have a real node for that space yet, it should under the root node for the time being. Other architectures seem to sometimes use a `/clocks` node, where this could be put together with the reference oscillator.

Reply:
fixed in upcoming version